VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingPanelBase.cs

```diff
@@ -269,11 +269,11 @@ protected override Size MeasureOverride(Size availableSize)
             {
                 /* Sometimes the scrollbar gets hidden without any reason, to prevent
                  * a layout circle, return without any recalculation. */
-                if (ScrollOwner.ComputedVerticalScrollBarVisibility != Visibility.Visible && ViewportHeight < ExtentHeight)
+                if (ScrollOwner?.ComputedVerticalScrollBarVisibility != Visibility.Visible && ViewportHeight < ExtentHeight)
                 {
                     return availableSize;
                 }
-                if (ScrollOwner.ComputedHorizontalScrollBarVisibility != Visibility.Visible && ViewportWidth < ExtentWidth)
+                if (ScrollOwner?.ComputedHorizontalScrollBarVisibility != Visibility.Visible && ViewportWidth < ExtentWidth)
```

Comment:
This was already fixed in https://github.com/sbaeumlisberger/VirtualizingWrapPanel/commit/bcb5ac6a951ae4fd2dae4f5645d60e0a2fc8f02d